tests/network/vmi_multus.go

```diff
@@ -91,6 +91,7 @@ const (
 )
 
 const (
+	vlanId                            = 100
```

Comment:
moving this constant away from it's usage might be less readable. vlanId is only used once, so I think you should consider not use it as global.